hips/draw/simple.py

```diff
@@ -117,6 +117,38 @@ def _fetch_tiles(self) -> HipsTile:
             tile = HipsTile.fetch(tile_meta, url)
             yield tile
 
+    @staticmethod
+    def _measure_tile_shape(corners: tuple) -> List[list]:
+        """Compute tile edges and diagonals."""
+
+        def compute_distance(i: list, j: list) -> float:
+            """Compute distance between two points."""
+            return np.sqrt((i[0] - i[1]) ** 2 + (j[0] - y[1]) ** 2)
+
+        x, y = corners
+        edges = []
+        for i in range(len(x)):
+            if i == 3:
+                edges.append(compute_distance([x[i], x[0]], [y[i], y[0]]))
+            else:
+                edges.append(compute_distance([x[i + 1], x[i]], [y[i + 1], y[i]]))
+
+        diagonals = []
+        diagonals.append(compute_distance([x[2], x[0]], [y[2], y[0]]))
+        diagonals.append(compute_distance([x[3], x[1]], [y[3], y[1]]))
+
+        ratio = float(np.min(diagonals) / np.max(diagonals))
+
+        return [edges, diagonals, ratio]
+
+    def is_tile_distorted(self, corners: tuple) -> bool:
+        """Implement tile splitting criteria as mentioned in :ref:`drawing_algo` page."""
+        edges, diagonals, ratio = self._measure_tile_shape(corners)
+
+        return any(i > 300 for i in edges) or \
```

Comment:
I think I would find `max(edge_lengths) > 300` and equivalently for diagonals easier to read.